runner/diff.go

```diff
@@ -195,3 +196,25 @@ func diffDatabaseStats(new state.PostgresDatabaseStatsMap, prev state.PostgresDa
 	}
 	return
 }
+
+func diffServerIoStats(new state.PostgresServerIoStatsMap, prev state.PostgresServerIoStatsMap) (diff state.DiffedPostgresServerIoStatsMap) {
+	followUpRun := len(prev) > 0
+
+	diff = make(state.DiffedPostgresServerIoStatsMap)
+	for k, stats := range new {
+		var s state.DiffedPostgresServerIoStats
+		prevStats, exists := prev[k]
+		if exists {
+			s = stats.DiffSince(prevStats)
+		} else if followUpRun { // New since the last run
+			s = stats.DiffSince(state.PostgresServerIoStats{})
+		}
+		// Skip over empty diffs (which can occur either because there was no activity, or for fixed entries that never saw activity)
+		if s.Reads != 0 || s.Writes != 0 || s.Extends != 0 ||
+			s.Evictions != 0 || s.Reuses != 0 || s.Fsyncs != 0 {
```

Comment:
Not a big deal, but we could add Writebacks and `Hits` here as well (it seems very unlikely that that'd be the only activity on a system, but easier to be consistent here for a future reader of the code).